dra7xx.mk

```diff
@@ -0,0 +1,145 @@
+###############################################################################
+# Following variables defines how the NS_USER (Non Secure User - Client
+# Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
+# S_USER (Secure User - TA) are compiled
+###############################################################################
+override COMPILE_NS_USER   := 32
+override COMPILE_NS_KERNEL := 32
+override COMPILE_S_USER    := 32
+override COMPILE_S_KERNEL  := 32
+
+###############################################################################
+# Includes
+###############################################################################
+-include common.mk
+
+###############################################################################
+# Paths to git projects and various binaries
+###############################################################################
+U-BOOT_PATH     ?= $(ROOT)/u-boot
+LINUX_IMAGE     ?= $(LINUX_PATH)/arch/arm/boot/zImage
+STAGEING_AREA   ?= $(BUILD_PATH)/dra7xx
```

Comment:
I think it would be better to use `$(ROOT)/out/dra7xx` as staging area. Speaking about that word, should it be "staging" or "stageing"? I think I'm used to see "staging".